EduID/Utilities/Constants.swift

```diff
@@ -52,10 +45,15 @@ enum Constants {
     enum KeyChain {
         static let keyPrefix = Bundle.main.bundleIdentifier ?? "nl.eduid"
         static let challenge = "Challenge"
+        static let oidAuthState = "EDU_ID_OIDAuthState"
```

Comment:
I explicitly removed these, why are they back?

Reply:
I must have used an old branch?!

Reply:
A rebase should fix that then